atlp-rwanda/e-comm-team-techsmith-fn
# #185218937: A buyer should be able to add items to their wishlist

## What does this PR do?

This PR it will be Implementing buyer should be able to add items to their wishlist

## Description of Task to be completed?

- Adding Items into Wishlist

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ft-buyer-can-add-items-in-whishlist#185218937`

- Install node packages that the application depends on: `npm i`


- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?

- [#185218937](https://www.pivotaltracker.com/story/show/185218937)


## Screenshots (if appropriate)
<img width="1439" alt="Screenshot 2023-06-23 at 02 29 50" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/64655121/b5f9afe7-aac0-4ab6-a245-73ff5e21f992">

<img width="1440" alt="Screenshot 2023-06-23 at 07 41 35" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/64655121/a65a6f9d-1181-4c7f-85e2-c8da685fa3c1">

https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/64655121/2d49a5e3-0c39-48e0-a51b-b11864d5c01a


<img width="1433" alt="Screenshot 2023-06-23 at 07 37 48" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/64655121/4a4196cd-f5c3-452e-84ed-1131b4e6bf49">
